geps/gep-713/index.md

```diff
@@ -1,1157 +1,985 @@
 
-##### Application Developer Discoverability
+## End-to-end examples
 
-How does Ana, or any Application Developer who owns one or more Route objects know
-that their object is affected by Policy, which Policy is affecting it, and what
-the content of the Policy is?
+This section presents a series of synthetic examples of applications of metaresources for different kinds of topologies and contexts.
 
-The best outcome is that Ana needs to look only at a specific route to know what
-Policy settings are being applied to that Route, and where they come from.
-However, some of the other problems below make it very difficult to achieve this.
+In all cases, the background of targetable object kinds is assumed to be a hierarchy of network resource kinds `Gateway` (`g`) > `Route` (`r`) > `Backend` (`b`), where `Gateway` is the least specific kind (instances denoted “`gX`”) and `Backend` is the most specific kind (instances denoted “`bX`”).
 
-##### Policy Admin Discoverability
+Moreover, a `ColorPolicy` metaresource kind (or “policy kind”, equivalently) is defined however with variations in its semantics across examples to accommodate for each case. Instances of the `ColorPolicy` kind (denoted “`pX[spec]`” and referred to simply as “policies”) may target one or more kinds of targetable resources, depending on each example. A policy represents an intent to “color” the network traffic that flows through the portion of the network corresponding to the target with a given color or color set that is specified in the policy.
 
-How does the Policy Admin know what Policy is applied where, and what the content
-of that Policy is?
-How do they validate that Policy is being used in ways acceptable to their organization?
-For any given Policy object, how do they know how many places it's being used?
+### Example 1. Direct Policy
 
-##### Cluster Admin Discoverability
+In this example, the `ColorPolicy` policy kind is defined as an instance of the Direct class of metaresources. Instances of the `ColorPolicy` kind in this example can only target `Backend` resources.
 
-The Cluster Admin has similar concerns to the Policy Admin, but with a focus on
-being able to determine what's relevant when something is broken.
+Given:
 
-How does the Cluster Admin know what Policy is applied where, and what the content
-of that Policy is?
+the following state of targetable resources:
 
-For any given Policy object, how do they know how many places it's being used?
+* `g1` > `r1` > `b1`
+* `g1` > `r2` > `b2`
+  and the following state of `ColorPolicy` (`p`) policies, where `pX[spec]` → `bX` denotes a policy `pX` attached to (“targeting”) a `Backend` resource `bX`, intending to augment `bX`‘s behavior with `spec`:
 
-#### Evaluating and Displaying Resultant Policy
+* `p1[color:red]` → `b1`
+* `p2[color:blue]` → `b1` (conflicting policy, `p2.creationTimestamp` > `p1.creationTimestamp`)
```

Comment:
Good example, but this should be a diagram either in addition to or instead of this description. In reading this, I personally need to make a diagram either in my head or on paper anyway, so including diagrams here would at least make it quicker for _me_ to parse these examples.

Reply:
Done. Please let me know if the one I added works for you.